src/views/token-swap/components/swap-button/swap-button.view.jsx

```diff
@@ -28,30 +27,26 @@ function SwapButton ({ quotes }) {
     return `${mins.toString().padStart(2, '0')}:${secs.toFixed(0).toString().padStart(2, '0')}`
   }
 
-  const buttonLabels = {
-    successful: `Swap ${msToTime(timeUntilValid)}`,
-    failure: 'Insufficient liquidity'
-  }
-
   return (
     <div className={classes.root}>
-      {['successful', 'failure'].includes(quotes.status) &&
+      {selectedQuote &&
```

Comment:
I think that this should be `countdown && ...` as we are now using countdown below instead the `selectedQuote` right?

Reply:
selectQuote covers more cases